lib/galaxy/authnz/psa_authnz.py

```diff
@@ -133,6 +135,7 @@ def _setup_idp(self, oidc_backend_config):
         self.config["KEY"] = oidc_backend_config.get("client_id")
         self.config["SECRET"] = oidc_backend_config.get("client_secret")
         self.config["TENANT_ID"] = oidc_backend_config.get("tenant_id")
+        self.config["OIDC_ENDPOINT"] = oidc_backend_config.get("oidc_endpoint")
```

Comment:
@bgruening Depending on what the logic in social core is this migh be the cause of https://github.com/galaxyproject/galaxy/issues/18816.
You could try something like
```
if oidc_backend_config.get("oidc_endpoint"):
    self.config["OIDC_ENDPOINT"] = oidc_backend_config["oidc_endpoint"]
```

Reply:
Yes, this fixed it for us!

Do you want me to create a PR against 24.1?

Reply:
Yes, that would be great!